java/integTest/src/test/java/glide/SharedCommandTests.java

```diff
@@ -6941,6 +6941,97 @@ public void xclaim_return_failures(BaseClient client) {
     //         assertInstanceOf(RequestException.class, executionException.getCause());
     //     }
 
+    @SneakyThrows
+    @ParameterizedTest(autoCloseArguments = false)
+    @MethodSource("getClients")
+    public void xautoclaim(BaseClient client) {
+        String minVersion = "6.2.0";
+        assumeTrue(
+                REDIS_VERSION.isGreaterThanOrEqualTo(minVersion),
+                "This feature added in redis " + minVersion);
+        boolean isVersion7OrAbove = REDIS_VERSION.isGreaterThanOrEqualTo("7.0.0");
+        String key = UUID.randomUUID().toString();
+        String groupName = UUID.randomUUID().toString();
+        String zeroStreamId = "0-0";
+        String consumer = UUID.randomUUID().toString();
+
+        // Add 4 stream entries for consumer
+        String streamid_0 = client.xadd(key, Map.of("f1", "v1" /*, "f2", "v2"*/)).get();
+        assertNotNull(streamid_0);
+        String streamid_1 = client.xadd(key, Map.of("field1", "value1")).get();
+        assertNotNull(streamid_1);
+        String streamid_2 = client.xadd(key, Map.of("field2", "value2")).get();
+        assertNotNull(streamid_2);
+        String streamid_3 = client.xadd(key, Map.of("field3", "value3")).get();
+        assertNotNull(streamid_3);
+
+        // create group and consumer for the group
```

Comment:
Do you cover binary APIs in the tests?

Reply:
Not here, but we have unit tests in `RedisClientTest.java`. Will that be enough?

Reply:
No, tnose are UT which execute no real command

Reply:
added